include/DecisionTree.hpp

```diff
@@ -27,7 +32,14 @@ struct BestFeatureResult {
 class DecisionTreeNode;
 class DecisionTree : public std::enable_shared_from_this<DecisionTree> {
   public:
+
+    // MARK: Discuss if these need to be private or can stay public
```

Comment:
should be completed before merge or after?

Reply:
prolly before, but since Eli merged we ball ig